<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilePartEditorWidget</class>
 <widget class="QWidget" name="FilePartEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit File Part</string>
  </property>
  <widget class="QLineEdit" name="file_path_linedit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Select file or folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="file_path_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>File Path:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="folder_path_button">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Folder</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="file_part_relative_to_scenario_folder_checkbox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Relative to Scenario Folder</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
